ui/app/templates/rule_modal.html

```diff
@@ -187,6 +187,10 @@ <h3 class="modal-title" ng-show="!is_edit && is_duplicate">Duplicate Existing Ru
     <input type="hidden" name="data_version" ng-model="rule.data_version">
   </form>
 
+  <div ng-show="ruleSignoffsRequired().length">
+    Making this change requires signoffs and therefore must be a scheduled change.
+  </div>
```

Comment:
It looks like this never ends up getting shown, because you can't open modals like this with the buttons disabled. Can you put it in the scheduled change modal instead? I think it would also be good to list out the required signoffs in there.

Reply:
This can get shown if, for example, there are required signoffs on Firefox Nightly, and you click "Update" on Firefox Aurora, and change the channel to Nightly.

Reply:
Ah, indeed! Perhaps we should disable the "Save Changes" button in that case, too, since the request would just return a 400 anyways?

Reply:
And styling the text a little differently would be nice, too. Maybe the same as the error div just beneath it?

Reply:
That sounds good, thanks. Do you still want the signoffs listed? Per https://bugzilla.mozilla.org/show_bug.cgi?id=1355074, you wrote "I don't think we need to show the full details of who needs to signoff, just some sort of "signoff required" indicator."

Reply:
Heh, apparently I'm indecisive. We may as well show them if we have them - final answer!